src/bank.rs

```diff
@@ -591,13 +566,9 @@ impl Bank {
 
         // Call the contract method
         // It's up to the contract to implement its own rules on moving funds
+        // TODO budget, storage, tictactoes can move to native dynamic
```

Comment:
nit: no need to add this line, file an issue instead :)

Reply:
Yup